cinema/serializers.py

```diff
@@ -22,3 +22,21 @@ def update(self, instance, validated_data):
         instance.save()
 
         return instance
+
+
+class ActorSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Actor
+        fields = "__all__"
```

Comment:
A small note that between class name and the first subclass/method should be 1 empty line